tests/unit/test_logging.py

```diff
@@ -245,3 +246,17 @@ def test_root_logger_configuration(root_logger_reset):  # noqa: F811
     # Now perform check to see if RecordCompiler was actually added to
     assert level == 20  # INFO equates to 20 by default
     assert any(isinstance(handler, RecordCompiler) for handler in root_handlers)
+
+
+def test_secrets_logging_config(root_logger_reset, expect_json_log):  # noqa: F811
+    """Make sure secret values are not revealed when logging the configuration."""
+
+    class TestConfig(LoggingConfig):
+        secret_bytes: SecretBytes = Field(default=b"test")
+        secret_str: SecretStr = Field(default="test")
```

Comment:
But why 'privy'?

Reply:
Just something unique that does not appear elsewhere in the log line. "secret" is already in the name of the property. Maybe "test" could work as well but I thought it could appear via the logger name or something else.